.github/ISSUE_TEMPLATE/1_bug_report.md

```diff
@@ -1,3 +1,5 @@
+# SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
```

Comment:
this is not markdown comments and we can not add stuff in here iirc.

So needs to go to the REUSE.toml